database-mongoose/seeder.js

```diff
@@ -1,85 +1,86 @@
+      // product properties
+      var prodId = 1000 + i;
+      var prodColor = faker.commerce.color();
+      var prodConfig = faker.lorem.words();
+      var prodCategory = 'Amazon Devices';
 
-    let testData =
-    {
-      reviewId: count,
-      productId: prodId,
-      color: prodColor,
-      configuration: prodConfig,
-      //isBestSeller: true,
-      category: prodCategory,
-      customerId: custId,
-      customerName: custName,
-      cutomerCountry: custCountry,
-      profilePicUrl: custProfile,
-      title: faker.lorem.words(),
-      description: faker.lorem.sentence(),
-      rating: randomRating,
-      isVerifiedPurchase: faker.random.boolean(),
-      isHelpfulCount: faker.random.number(),
-      imageUrlId: randomProdImages,
-      reviewDate: faker.date.past(),
-      easeToUse: Math.floor(Math.random() * (5 - 1 + 1) + 1),
-      voiceRecognition: Math.floor(Math.random() * (5 - 1 + 1) + 1),
-      techSupport: Math.floor(Math.random() * (5 - 1 + 1) + 1),
-      valueForMoney: Math.floor(Math.random() * (5 - 1 + 1) + 1),
-      qualityOfMaterial: Math.floor(Math.random() * (5 - 1 + 1) + 1),
-      batteryLife: Math.floor(Math.random() * (5 - 1 + 1) + 1)
-    };
+      let testData =
+      {
+        reviewId: count,
+        productId: prodId,
+        color: prodColor,
+        configuration: prodConfig,
+        //isBestSeller: true,
+        category: prodCategory,
+        customerId: custId,
+        customerName: custName,
+        customerCountry: custCountry,
+        profilePicUrl: custProfile,
+        title: faker.lorem.words(),
+        description: faker.lorem.sentence(),
+        rating: randomRating,
+        isVerifiedPurchase: faker.random.boolean(),
+        isHelpfulCount: faker.random.number(),
+        imageUrls: randomProdImages,
+        reviewDate: faker.date.past(),
+        easeToUse: Math.floor(Math.random() * (5 - 1 + 1) + 1),
+        voiceRecognition: Math.floor(Math.random() * (5 - 1 + 1) + 1),
+        techSupport: Math.floor(Math.random() * (5 - 1 + 1) + 1),
+        valueForMoney: Math.floor(Math.random() * (5 - 1 + 1) + 1),
+        qualityOfMaterial: Math.floor(Math.random() * (5 - 1 + 1) + 1),
+        batteryLife: Math.floor(Math.random() * (5 - 1 + 1) + 1)
+      };
 
-    // console.log(testData);
-    // console.log("==============================");
-    // console.log('Review Id: ', testData.reviewId);
-    // console.log('Product Id: ', testData.productId);
-    // console.log('Color Id: ', testData.color);
-    // console.log('Customer Id: ', testData.customerId);
-    // console.log('Customer Name : ', testData.customerName);
-    allTestData.push(testData);
+      allTestData.push(testData);
 
+    }
   }
+  return allTestData;
+
 }
-//console.log('allTestData : ', allTestData)
-dbModel.insertSeedData(allTestData);
+
+var testData = generateSeedData(11, 100);//11 customers, 100 products
+//console.log(testData);
+ReviewService.insertSeedData(testData);
```

Comment:
This looks good. Looks intricate but looks like you made it work! Something minor I noticed was some left over commented out console.logs

Reply:
Yes, will be cleaning up comments for next PR.